web/pom.xml

```diff
@@ -80,7 +80,7 @@
         <profile>
             <id>curate</id>
             <properties>
-                <frontend.version>0.1.0</frontend.version>
+                <frontend.version>5e87fed167</frontend.version>
```

Comment:
Should we commit front-end version every time?

Reply:
@victoria34 this is for legacy public frontend which we still sometime use in order to avoid deploy two applications. You know we now separate the public website from core, so in order to deploy a core with a frontend, we rely on this legacy app which hosts on jitpack. We don't tag the version for legacy anymore, so we use the commit.

Reply:
I see. Thanks for the explanation.